test/fluree/db/ledger/api/downloaded.clj

```diff
@@ -276,6 +294,26 @@
 
       (is (every? boolean (map #(collection-names %) ["_rule" "_fn" "_predicate" "_setting" "_auth" "_user" "_shard" "_tag" "_role" "_collection"]))))))
 
+(deftest sign-all-collections-graphql
+  (testing "sign a query for all collections through the graphql endpoint"
+    (let [private-key         (slurp "default-private-key.txt")
+          qry-str             (json/stringify {:query "{  graph {  _collection (sort: {predicate: \"name\", order: ASC}) { _id name spec version doc}}}"})
+          request             {:headers {"content-type" "application/json"}
+                               :body    qry-str}
+          q-endpoint          (str endpoint-url "graphql")
+          signed-req          (http-signatures/sign-request :post q-endpoint request private-key)
+          resp                @(http/post q-endpoint signed-req)
+          body                (-> resp :body bs/to-string json/parse)
+          collection-names    (into #{} (map #(:name %) (-> body :data :_collection)))]
+
+      ; Are the collection keys what we expect?
+      (is (map #(#{:doc :version :spec :name :_id} %) body))
```

Comment:
Another good candidate for `contains-many?`?

Reply:
✅